.gitignore

```diff
@@ -1,3 +1,2 @@
-.idea/*
-*.iml
-target/*
+# Ignore log files
+*.log
```

Comment:
Why do you remove the previous .gitignore configuration? 
Now you push a lot of redundant files to GH. Let’s fix it ;)